src/Microsoft.Restier.Core/Conventions/ConventionBasedChangeSetItemFilter.cs

```diff
@@ -86,51 +86,55 @@ private Task InvokeProcessorMethodAsync(SubmitContext context, ChangeSetItem ite
             var dataModification = (DataModificationItem)item;
             var expectedMethodName = ConventionBasedMethodNameFactory.GetEntitySetMethodName(dataModification, pipelineState);
             var expectedMethod = targetApiType.GetQualifiedMethod(expectedMethodName);
-            if (!IsUsable(expectedMethod))
+
+            if (expectedMethod == null)
             {
-                if (expectedMethod != null)
-                {
-                    Trace.WriteLine($"Restier Filter found '{expectedMethodName}' but it is unaccessible due to its protection level. Your method will not be called until you change it to 'protected internal'.");
-                }
-                else
+                var actualMethodName = expectedMethodName.Replace(dataModification.ExpectedResourceType.Name, dataModification.ResourceSetName);
```

Comment:
It's easier to see in the entire diff, but this is an existing check that is just rewritten a bit to add more tracing when a method does not conform to the standards as specified by the docs. There were quite a few cases that I could produce for filter methods that were "Unusable" but did not generate a meaningful trace message.

Reply:
Do you have a case as an example? I just want to understand because I think the trace message could use some clarification.

Reply:
Which one? `Restier Filter expected'{expectedMethodName}' but found '{actualMethodName}`?

That one was probably a breaking change (since it had "BREAKING: " in front of it), but essentially it seems that the filter methods used to use the name of the DBSet property on the context (e.g. People), whereas it will use the name of the Resource type now (e.g. Person). Again, this is not my trace message, I just rewrote the logic and added the other ones.

Reply:
OK, gotcha.